chat-widget/src/plugins/newMessageEventHandler.ts

```diff
@@ -77,6 +77,12 @@ export const createOnNewAdapterActivityHandler = (chatId: string, userId: string
                         Description: "New message received",
                         Data: payload
                     });
+                } else {
+                    TelemetryHelper.logActionEvent(LogLevel.INFO, {
```

Comment:
This is going to log all messages every 30 seconds. Please check if this will overflow out database

Reply:
Updated to have only one historymessage
![image](https://user-images.githubusercontent.com/45749980/220534397-edb08e45-2b77-4bc1-a6d1-1d2091899c62.png)